src/MBP_UserImport_BaseSource.php

```diff
@@ -70,6 +70,19 @@ public function __construct($message) {
   abstract protected function setKeys();
   
   /**
+   * Logic to determine if data from row in CSV file can be processed.
+   *
+   * @param array $data
+   *   Values to test.
+   */
+  abstract public function canProcess($data);
+
+  /**
+   * Assign columns specific to the source to common columns expected by the consumer.
+   */
+  abstract public function setter(&$data);
```

Comment:
Good point but `setter()` has been used at the abstract class level throughout the Message Broker system. We'll have do a bunch of renaming in many places throughout the system. I'm not sure the effort is worth the clarity.

Lets make a point of using an alternative name for the next version of `MB_Toolbox -> 0.10.x` when we're feeling ambitious.